Makefile

```diff
@@ -374,6 +374,11 @@ out/test-report.json: $(SOURCE_FILES) $(GOTEST_FILES)
 	-coverprofile=out/coverage.out -json > out/test-report.json
 out/coverage.out: out/test-report.json
 
+# Generate go test report (from gotest) as a a HTML page
+test-report.html: out/test-report.json
```

Comment:
could we call it unit-test-report.html ?

Reply:
I wanna ensure this wont confuse the users of 
```
$ make help | grep html
html_report                    Generate HTML  report out of the last ran integration test logs.
```

Reply:
or you could make the "html_report" convert all tests to html (including last integraiton and unit test)
that way we dont have to remember which one to call...

jsut ask makefile to give us html for all tests in out/unit-test.html

Reply:
Renamed and moved to out